### ISSUE TITLE
[Question] How to prioritize datapack over eachother?

### ISSUE THREAD

**Issue by ardissaps** [Declipsonator/Global-Datapack#4]
Is it possible to priority some datapack like resource pack?
And will you support of management required global resourcepack that sometimes required for some datapack?

**Comment by Declipsonator** [comment]
Here's a website which explains the first question:
https://nodecraft.com/support/games/minecraft/commands-for-minecraft-datapacks

I'm not quite sure what you're asking in the second question can you rephrase it a little?
